templates/components/form/header.html.twig

```diff
@@ -86,7 +87,7 @@
                   </div>
                {% endif %}
                <span>
-               {{ nametype }}
+               {{ nametype }} - {{ friendlyname }}
```

Comment:
I'm currently asking myself if the nametype is really necessary.
We have the information from the tab name

Reply:
Not a big deal but if I had to choose I'd keep the nametype.
It makes sense to me that the most important header should contain the full information (type + name).